public/js/main.js

```diff
@@ -392,13 +392,37 @@ function updateAlerts() {
   }
 }
 
+function updateSortie() {
+  var sortie = worldState.sortie;
+  
+  if (sortie.length !== 0) {
+	$('#sortietitle').hide();
+	
+	$('#sortieBoss').html(sortie.boss);
+	$('#sortieFaction').html(sortie.faction);
+	$('#sortieList').find('.variant').remove();
```

Comment:
Are there any advantages to remove and add back information that may not be changed?

Reply:
Not really, this and the other comment below was why I was didn't want to commit quite yet but Tobi insisted that I should commit it.

Reply:
:( _i'm sorry_

Reply:
i get excited and like to see what ppl are working on

Reply:
Eh, it's fine. It's just that I was in the "get it working" phase. I will continue to refine it.

Reply:
Sorry, I will review the entire thing on your cue.
For now, ignore the things mentioned here as you mentioned that it is still work in progress.